dask_geomodeling/tests/test_geometry.py

```diff
@@ -1125,6 +1149,12 @@ def test_set_series_overwrite(self):
         assert_almost_equal(added_values, [i * 3 for i in range(self.N)])
         self.assertSetEqual({"geometry", "col_1"}, source.columns)
 
+    def test_set_series_with_nan(self):
+        source = geometry.SetSeriesBlock(self.source1, "added", self.source2["col_4"])
+        data = source.get_data(**self.request)
+        added_values = data["features"]["added"].values
+        assert np.isnan(added_values[::2]).all()
```

Comment:
Just curious, why were you unsure about this, so that it deserves a test on its own?

Reply:
No, this was my first guess were the bug was. But it wasn't there. It doesn't really deserve testing indeed.